### PR TITLE
Keep `end port` when updating config

### PR DESCRIPTION
**Pull request by banasiakadam60**
Signed-off-by: Adam Banasiak <banasiakadam60@gmail.com>

### CODE REVIEW

**Comment by banasiakadam60** [comment]
Fix for: #1230

**Review by ggruszczynski — DISMISSED** [review]
ok

**Comment by banasiakadam60** [comment]
@badb Can you check this PR?

File: apps/blender/resources/scripts/docker_blendertask.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def format_blender_render_cmd(outfilebasename, scene_file, script_file,
         "-o", "{}/{}_{}".format(OUTPUT_DIR, outfilebasename, start_task),
         "-noaudio",
         "-F", "{}".format(output_format.upper()),
-        "-t", "{}".format(cpu_count()),
+        "-t", "{}".format(min(32, cpu_count())),
```
Comment (badb) [inline apps/blender/resources/scripts/docker_blendertask.py]: Can you make this one commen function `get_cpu_count` that is in `golem.core.common`? 
Also if it VirtualBox restriction that it should be only used on Windows / Mac?

**Review by badb — DISMISSED** [review]
Can you please separte those two issues?

**Comment by banasiakadam60** [comment]
OK, `max_cpu` moved to [another PR](https://github.com/golemfactory/golem/pull/1332)

File: apps/blender/resources/scripts/docker_blendertask.py (hunk 1 above)
Comment (banasiakadam60) [reply apps/blender/resources/scripts/docker_blendertask.py]: ## Windows

From [VBox Manual](http://download.virtualbox.org/virtualbox/5.1.26/UserManual.pdf)
On page 14
```
– Guest multiprocessing (SMP). VirtualBox can present up to 32 virtual CPUs to each
virtual machine, irrespective of how many CPU cores are physically present on your
host.
```

Section `3.4.2` (page 51)
```
Starting with version 3.0, VirtualBox supports symmetrical multiprocessing (SMP)
and can present up to 32 virtual CPU cores to each virtual machine.
```

## MacOS 

`xhyve` has right now limitation for max [16 cores](https://github.com/mist64/xhyve/blob/1f1dbe3059904f885e4ab2b3328f4bb350ea5c37/include/xhyve/vmm/vmm_common.h#L36)

## Linux

Happy - no limitations

File: golem/interface/client/settings.py [hunk 2]
```diff
@@ -187,8 +187,8 @@ class Settings(object):
     @command(arguments=(basic, provider, requestor),
              help="Show current settings")
     def show(self, basic, provider, requestor):
-
-        config = sync_wait(Settings.client.get_settings())
+        settings = yield Settings.client.get_settings()
+        config = settings
```
Comment (mfranciszkiewicz) [inline golem/interface/client/settings.py]: `config = sync_wait(Settings.client.get_settings())`

**Comment by banasiakadam60** [comment]
@mfranciszkiewicz @badb Can you review it?

File: golem/clientconfigdescriptor.py [hunk 3]
```diff
@@ -8,54 +8,75 @@ class ClientConfigDescriptor(object):
...
+                                     P2P_SESSION_TIMEOUT, TASK_SESSION_TIMEOUT,
+                                     RESOURCE_SESSION_TIMEOUT,
+                                     ESTIMATED_DEFAULT, NODE_SNAPSHOT_INTERVAL,
+                                     NETWORK_CHECK_INTERVAL,
+                                     MAX_SENDING_DELAY, MIN_CPU_CORES,
+                                     MIN_DISK_SPACE, MIN_MEMORY_SIZE,
+                                     DEFAULT_HARDWARE_PRESET_NAME,
+                                     REQUESTING_TRUST, COMPUTING_TRUST,
+                                     MIN_PRICE,
+                                     MAX_PRICE, ACCEPT_TASKS)
```
Comment (jiivan) [inline golem/clientconfigdescriptor.py:30]: Maybe just importing `from golem import appconfig` will prevent cluttering of namespace and simplify code?

**Review by jiivan — APPROVED** [review]
LGTM except the namespace cluttering with import in `golem/clientconfigdescriptor.py`

**Comment by badb** [comment]
Golem crashes after configuration changes

**Comment by badb** [comment]
No longer needed, because old interface was removed anyway.

**Pull request closed without merging**